package.json

```diff
@@ -43,6 +43,7 @@
     "react-native-gesture-handler": "~2.5.0",
     "react-native-keyboard-aware-scroll-view": "^0.9.5",
     "react-native-localize": "^2.2.4",
+    "react-native-material-textfield": "^0.16.1",
```

Comment:
i don't see this being used, so you can remove it. should probably just call yarn remove on this package